packages/explat-client/src/internal/test/experiment-assignment-store.ts

```diff
@@ -1,37 +1,46 @@
+/**
+ * @jest-environment jsdom
+ */
+
 /**
  * Internal dependencies
  */
-import ExperimentAssignmentStore from '../experiment-assignment-store';
+import {
+	retrieveExperimentAssignment,
+	storeExperimentAssignment,
+} from '../experiment-assignment-store';
 import { validExperimentAssignment, validFallbackExperimentAssignment } from '../test-common';
 
+beforeEach( () => {
+	window.localStorage.clear();
+} );
+
 describe( 'experiment-assignment-store', () => {
 	it( 'should save and retrieve valid ExperimentAssignments', () => {
-		const experimentAssignmentStore = new ExperimentAssignmentStore();
-		expect( experimentAssignmentStore.retrieve( validExperimentAssignment.experimentName ) ).toBe(
+		expect( retrieveExperimentAssignment( validExperimentAssignment.experimentName ) ).toBe(
 			undefined
 		);
-		experimentAssignmentStore.store( validExperimentAssignment );
-		expect(
-			experimentAssignmentStore.retrieve( validExperimentAssignment.experimentName )
-		).toEqual( validExperimentAssignment );
+		storeExperimentAssignment( validExperimentAssignment );
+		expect( retrieveExperimentAssignment( validExperimentAssignment.experimentName ) ).toEqual(
```

Comment:
Should this be `toStrictEqual` as below?

```suggestion
		expect( retrieveExperimentAssignment( validExperimentAssignment.experimentName ) ).toStrictEqual(
```